src/app/workorder/workorder.js

```diff
@@ -176,6 +176,8 @@ angular.module('app.workorder', [
   self.workers = workers;
 
   mediator.subscribeForScope('wfm:workorder:created', $scope, function(workorder) {
+    //remove generated angular variables
+    workorder = JSON.parse(angular.toJson(workorder));
```

Comment:
Same as above, this is published in the https://github.com/feedhenry-raincatcher/raincatcher-workorder/blob/master/lib/angular/directive.js#L150 module.

It would be better to fix it there.